src/components/settings/MergeConfirmationModal.tsx

```
@@ -0,0 +1,133 @@
+                            <li>The account <span className="font-bold">{secondaryEmail}</span> will be <span className="font-bold text-red-600">permanently deleted</span>.</li>
+                            <li>This action cannot be undone.</li>
+                        </ul>
+                    </div>
+
+                    {error && (
+                        <div className="p-3 bg-red-50 dark:bg-red-900/20 border border-red-200 dark:border-red-800 text-red-600 dark:text-red-400 rounded-lg text-sm">
+                            {error}
+                        </div>
+                    )}
+
+                    <div>
```

ESLint

[error] Replace `····················` with `··········`

(prettier/prettier)